### PR TITLE
add a few synonyms

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: geocoder_tester/world/france/test_synonymes.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_city
+2 rue du 10e niort,,,4,,2,Rue du Dixième,Niort
+4 rue 11 novembre Frénouville,,,4,,4,Rue du Onze Novembre,Frénouville
```
Comment (antoine-de) [inline geocoder_tester/world/france/test_synonymes.csv]: when I search for this one in all the geocoders I find `rue 11 novembre`

File: geocoder_tester/world/france/test_synonymes.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+1 place du cc Lomprez,,,4,,1,Place du Centre Commercial Lomprez,Valenciennes
+2 Place du 12 Mai Figeac,,,4,,2,Place du Douze Mai,Figeac
+4 Place du 1er Mai,,,4,,4,Place du Premier Mai,Ennezat
```
Comment (antoine-de) [inline geocoder_tester/world/france/test_synonymes.csv]: same here I find `1er` in autocomplete comparator

File: geocoder_tester/world/france/test_synonymes.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_city
+2 rue du 10e niort,,,4,,2,Rue du Dixième,Niort
```
Comment (antoine-de) [inline geocoder_tester/world/france/test_synonymes.csv]: this one is tricky, bragi match `Paris 10e Arrondissement` :laughing: 

only addock manage it.
nice, but difficult test!

File: geocoder_tester/world/italy/test_synonyms.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+query,lon,lat,limit,lang,expected_name,expected_housenumber,expected_street,expected_city
+C.po san polo venezia,,,4,it,,,Campo San Polo,Venezia
+L.go Guglielmo Henzen Roma,,,4,it,,,Largo Guglielmo Henzen,Roma
+Via del fontile Anagnino Roma,,,4,it,,,Via del Fontanile Anagnino,Roma
```
Comment (antoine-de) [inline geocoder_tester/world/italy/test_synonyms.csv]: isn't it more a typo than a synomyme ? (I'm not an italian expert :stuck_out_tongue_winking_eye: )

File: geocoder_tester/world/france/test_synonymes.csv (hunk 1 above)
Comment (nlehuby) [reply geocoder_tester/world/france/test_synonymes.csv]: the expected answer is this one : https://www.openstreetmap.org/way/333816885

File: geocoder_tester/world/italy/test_synonyms.csv (hunk 4 above)
Comment (nlehuby) [reply geocoder_tester/world/italy/test_synonyms.csv]: found it https://github.com/openvenues/libpostal/blob/01afbf80efd9baf2adbe62c97901b4fd98f8854b/resources/dictionaries/it/synonyms.txt#L41

**Pull request merged**